### PR TITLE
vis: add utility accessors for custom visualisations

### PR DESCRIPTION
**Pull request by sondove**
Novem offers custom visuals where the user can specify their own css and js code. Here we add some convenient utility functions that are consistent with how other novem api cases such as `colors` and `shared` are used.

Fixes novem-code/novem-python#43

### CODE REVIEW

File: novem/vis/custom.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from novem.vis.plot import Plot
+
+
+class NovemCustom:
+    def __init__(self, api: "Plot") -> None:
+        self.api: "Plot" = api
+
```
Comment (bjornars) [inline novem/vis/custom.py:10]: I think this is a nice style:
```python
from dataclass import dataclass

@dataclass(frozen=True)
class NovemCustom:
    api: "Plot"

   <functions go here>
```
(not tested)

File: novem/vis/custom.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @property
+    def js(self) -> str:
+        """
+        get the custom JS
+        """
```
Comment (bjornars) [inline novem/vis/custom.py:15]: these comments are pointless, 🔥 them

Comment (bjornars) [inline novem/vis/custom.py:15]: these comments are pointless, 🔥 them

**Review by bjornars — APPROVED** [review]

File: novem/vis/custom.py (hunk 1 above)
Comment (sondove) [reply novem/vis/custom.py]: Nice, incorporated

**Pull request merged**